prow/config/config.go

```diff
@@ -270,21 +246,6 @@ func parseConfig(c *Config) error {
 		return fmt.Errorf("plank has invalid max_concurrency (%d), it needs to be a non-negative number", c.Plank.MaxConcurrency)
 	}
 
```

Comment:
The validation for the plank config looks identical to that of the jenkins-operator.
Can you move this to a `ValidateController` function that can be used on `c.Plank` here and also called from `jenkins/config.go`?